lib/platform/common.ts

```diff
@@ -123,7 +123,8 @@ export interface Platform {
     title: string,
     body: string,
     once?: boolean,
-    shouldReopen?: boolean
+    shouldReopen?: boolean,
```

Comment:
If you pass `null` to `once` and `shouldReopen`, those types should be `boolean | null`. An alternative is passing `undefined` instead of `null`.